### DISCUSSION TITLE
Is there a way to do a conditonal upsert?

### DISCUSSION THREAD

**Discussion by slloyd88** [PostgREST/postgrest#2995] [Q&A] [answered]
I need a way to upsert ONLY WHEN the seq_field value in the payload is greater than the seq_field value already in the table.
Is there a way to do this?

**Comment by steve-chavez** [comment, marked as answer]
Only through a function for now https://postgrest.org/en/stable/references/api/stored_procedures.html.

After https://github.com/PostgREST/postgrest/issues/1959 is implemented it might be possible to do:

```
PATCH /items?seq_field=gt.$seq_field
{seq_field: ..}

or

PATCH /items?seq_field=$gt.seq_field
{seq_field: ..}
```
